docs/docs/extending-mdx.server.mdx

```diff
@@ -68,6 +68,9 @@ You can use this template:
 See also the [list of remark plugins][remark-plugins] and
 [list of rehype plugins][rehype-plugins].
 
+*   [`domdomegg/recma-mdx-displayname`](https://github.com/domdomegg/recma-mdx-displayname)
+    — add a `displayName` to MDXContent elements, to enable switching
+    on them in production
```

Comment:
```suggestion
    — add a `displayName` to `MDXContent` components, to enable switching
    on them in production
```

Good to use code for variable names, and these are components, not elements yet!

Otherwise looks good to me! 👍 Congrats on your plugin, looks good!